extension/src/popup/views/SendPayment/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+
+import { Switch } from "react-router-dom";
+import { PublicKeyRoute } from "popup/Router";
+import { ROUTES } from "popup/constants/routes";
+
+import {
+  SendAmount,
+  SendTo,
+  SendSettings,
+  SendConfirm,
+} from "popup/components/sendPayment";
+
+import { getAccountBalances } from "@shared/api/internal";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import { publicKeySelector } from "popup/ducks/accountServices";
+
+import { defaultAccountBalances } from "popup/views/Account";
+
+export const SendPayment = () => {
+  // keep state separate for now, combine later if needed
+  const [amount, setAmount] = useState("");
+  const [asset, setAsset] = useState("native");
+  const [destination, setDestination] = useState("");
+  const [transactionFee, setTransactionFee] = useState("");
+  const [memo, setMemo] = useState("");
+
+  const publicKey = useSelector(publicKeySelector);
+  const networkDetails = useSelector(settingsNetworkDetailsSelector);
+  const [accountBalances, setAccountBalances] = useState(
+    defaultAccountBalances,
+  );
+
+  useEffect(() => {
+    const fetchAccountBalances = async () => {
+      try {
+        const res = await getAccountBalances({
+          publicKey,
+          networkDetails,
+        });
+        setAccountBalances(res);
+      } catch (e) {
+        console.error(e);
+      }
+    };
+    fetchAccountBalances();
+  }, [publicKey, networkDetails]);
```

Comment:
Since we fetch fetchAccountBalances in `Account/index.tsx`, and we always end up on this View from Account, we can pass those fetched balances if use <Link state>:

https://v5.reactrouter.com/web/api/Link

Reply:
til!
https://github.com/stellar/freighter/pull/372/commits/26cd683656a1e493a5f9d7a1c45bba85ab708d9e

Reply:
note: had to stringify it or else React non-serialized data error pops up because of BigNumber balances